crowdfunding_compassion/forms/project_update_form.py

```diff
@@ -74,4 +78,27 @@ def form_before_create_or_update(self, values, extra_values):
         """ Dismiss any pending status message, to avoid multiple
         messages when multiple forms are present on same page.
         """
+        name = extra_values.get("name")
+        if name:
+            values["name"] = name
         self.o_request.website.get_status_message()
+
+    def form_after_create_or_update(self, values, extra_values):
+        if values.get("name") or values.get("description"):
+            # Notify responsible of the changes, for validation.
+            settings = self.env["res.config.settings"].sudo()
+            notify_ids = settings.get_param("new_participant_notify_ids")
+            if notify_ids:
+                user = self.env["res.partner"].sudo().browse(notify_ids[0][2]) \
+                    .mapped("user_ids")[:1]
+                self.main_object.sudo().activity_schedule(
+                    'mail.mail_activity_data_todo',
+                    summary="Verify project information.",
+                    note=f"{self.main_object.project_owner_id.name} updated the "
+                    f"name and description of the project. "
+                    f"Please check if the information is good enough.",
+                    user_id=user.id
```

Comment:
Are we sure that there will be a user in all cases (since we get the user using `:1` instead of simply `0`) ? I don't know if it would be overkill to put the `activity_schedule()` inside a `if user:`.

Reply:
If notify_ids is set there should be no reason that there is no user so I think we can keep it like that. Thanks for checking.